test/built-ins/Function/15.3.5.4_2-12gs.js

```diff
@@ -7,6 +7,7 @@ description: >
     Strict mode - checking access to non-strict function caller from
     non-strict function (eval includes strict directive prologue)
 flags: [noStrict]
+features: [caller]
```

Comment:
We have more files that use the `caller` property on the `Function` than the ones changed here. Is there a particular reason only these files were updated but the rest were left as they were? If there isn't, we should add the `caller` feature to the others also.